### PR TITLE
make table cell selector more specific

### PR DESCRIPTION
**Pull request by matthewcorreia-hitfox**

### CODE REVIEW

File: app/assets/stylesheets/default_theme/table.scss [hunk 1]
```diff
@@ -47,7 +47,8 @@
     }
   }
 
-  td {
+  td,
+  > tbody > tr > td {
```
Comment (dziemba) [inline app/assets/stylesheets/default_theme/table.scss]: Can you explain what this does?
 `td` is always a superset of `> tbody > tr > td`, so this doesn't change the behavior at all IMHO.

Comment (matthewcorreia-hitfox) [inline app/assets/stylesheets/default_theme/table.scss]: it needs to be more specific to override the default bootstrap styles, which are applied using the same rule (table>tbody>tr>td)

i included the more generic td selector to cover lazy/simple tables (tables without tbody)

Comment (dziemba) [inline app/assets/stylesheets/default_theme/table.scss]: Ah I see. :+1: 

Can you maybe add a comment to explain that in the source? It's not obviously by just looking at it.

Comment (matthewcorreia-hitfox) [inline app/assets/stylesheets/default_theme/table.scss]: ok

**Comment by dziemba** [comment]
:mushroom:

**Pull request merged**